docs/rules/no-pending-tests.md

```diff
@@ -0,0 +1,41 @@
+# Disallow Pending Tests (no-pending-tests)
+
+Mocha allows specification of pending tests, which represent tests that aren't yet implemented, but are intended to be implemented eventually. These are designated like a normal mocha test or suite, but with only the first argument provided (no callback for the actual implementation). For example: `describe('unimplemented bdd suite');`
```

Comment:
Oh I see -- just added a new commit to remove the handling for those. I guess I just assumed mocha supported pending suites.